gensim/models/base_any2vec.py

```diff
@@ -612,6 +614,7 @@ def __init__(self, sentences=None, input_streams=None, workers=3, vector_size=10
             logger.warning("consider setting layer size to a multiple of 4 for greater performance")
         self.alpha = float(alpha)
         self.window = int(window)
+        self.symmetric = bool(symmetric)
```

Comment:
need to modify `load` function (for backward compatibility reason) + don't forget about test

Reply:
also, if you put it here, don't forget to check how it works for all *2vec models (includes doc2vec)